ansible/roles/artifactory/tasks/main.yml

```diff
@@ -53,3 +58,6 @@
     dest: "/etc/profile.d/artifactory.sh"
     content: |
       export JFROG_HOME="/opt/jfrog"
+
+- name: Unconditionally reboot the machine with all defaults
+  reboot:
```

Comment:
This reboot serves no purpose at this stage of the process.

Executing this task will cause the Packer SSH connection to be closed but as it's the final step of provisioning, Packer will continue to shut down the instance and convert it to an AMI.

From what I can see, the best time for a reboot would be immediately after yum package upgrade to ensure that all subsequent steps are being performed on the most up to date packages.

However, as noted, the reboot is being performanced ungracefully and is terminating the SSH connection - you may wish to look in to an async wait task to accomplish the reboot.

Reply:
Moved the reboot module to be executed after the yum updates & completed via the implementation of the connect_timeout & reboot_timeout parameters from within the reboot module.
Instance gracefully rebooting & playbook continuing as required.